src/liballoc/boxed.rs

```diff
@@ -86,10 +86,13 @@ use core::raw::{TraitObject};
 #[lang = "exchange_heap"]
 #[unstable(feature = "box_heap",
            reason = "may be renamed; uncertain about custom allocator design")]
+#[allow(deprecated)]
```

Comment:
cc @pnkfelix -- can you comment on whether we may still want this constant with the upcoming box/allocator system?